src/Model/Photo.php

```diff
@@ -1095,7 +1095,7 @@ public static function upload(int $uid, array $files, string $album = '', string
 
 		$r = self::store($image, $user['uid'], 0, $resource_id, $filename, $album, 0, self::DEFAULT, $allow_cid, $allow_gid, $deny_cid, $deny_gid, $desc);
 		if (!$r) {
-			Logger::notice('Photo could not be stored');
+			Logger::warning('Photo could not be stored');
```

Comment:
Can we store some more context to better identify the root cause? Like resource id, uid,...

Reply:
Done